lib/shared/addon/components/form-pod-affinity-term-k8s/component.js

```diff
@@ -189,12 +186,12 @@ export default Component.extend({
       switch (val){
       case namespaceModes.ALL:
         set(this, 'namespaceSelector', {})
-        set(this, 'namespaces', null)
+        delete this.podAffintyTerm.namespaces
```

Comment:
Setting to null or deleting the `namespaces` key should have the same effect - I've updated this to delete the key to match the rke2 behavior.

Reply:
agree